test/user.js

```diff
@@ -18,6 +18,28 @@ const api = require('./lib/api.js');
 const prompt = require('../lib/prompt.js');
 const user = require('../lib/user.js');
 const util = require('../lib/util.js');
+const constants = require('./../lib/constants');
+
+const invalidEmail = 'invalid@example.com';
```

Comment:
Test constants/values should be moved to the appropriate `describe` block and initialized in a `before/beforeEach` block on the `this` namespace. An example:

```
describe('user', () => {
  beforeEach('credential setup', () => {
    this.invalidEmail = 'invalid@example.com';
    ...
  });
});
```